indico/modules/events/papers/controllers/management.py

```diff
@@ -219,3 +214,86 @@ def _process(self):
             flash(messages[role], 'success')
             return jsonify_data(html=_render_paper_dashboard(self.event))
         return jsonify_form(form)
+
+
+class RHManageReviewingQuestions(RHManagePapersBase):
+    def _process(self):
+        review_type = request.view_args['review_type']
+        if review_type == 'layout':
+            questions = self.event.cfp.layout_review_questions
+        else:
+            questions = self.event.cfp.content_review_questions
+
+        actions = {'create': 'papers.create_reviewing_question', 'edit': 'papers.edit_reviewing_question',
+                   'delete': 'papers.delete_reviewing_question', 'sort': 'papers.sort_reviewing_questions'}
+        args = {'create': {'review_type': review_type}, 'sort': {'review_type': review_type}}
+        return jsonify_template('events/reviewing_questions_management.html', event=self.event,
+                                reviewing_questions=questions, actions=actions,
+                                field_types=get_reviewing_field_types('papers'), args=args)
+
+
+class RHReviewingQuestionsActionsBase(RHManagePapersBase):
+    def _process_args(self):
+        RHManagePapersBase._process_args(self)
+        try:
+            self.review_type = request.view_args['review_type']
+        except KeyError:
+            raise NotFound
+
+
+class RHCreateReviewingQuestion(RHReviewingQuestionsActionsBase):
+    def _process_args(self):
+        RHReviewingQuestionsActionsBase._process_args(self)
+        try:
+            self.field_cls = get_reviewing_field_types('papers')[request.args['field_type']]
+        except KeyError:
+            raise NotFound
+
+    def _process(self):
+        form = self.field_cls.create_config_form()
+        if form.validate_on_submit():
+            new_question = create_reviewing_question(self.event, PaperReviewQuestion, self.field_cls, form,
+                                                     {'type': PaperReviewType.get(self.review_type)})
+            self.event.paper_review_questions.append(new_question)
+            logger.info("Paper reviewing question %s has been created by %s", new_question, session.user)
+            return jsonify_data(flash=False)
+        return jsonify_form(form, fields=getattr(form, '_order', None))
+
+
+class RHManageQuestionBase(RHReviewingQuestionsActionsBase):
+    def _process_args(self):
+        RHReviewingQuestionsActionsBase._process_args(self)
+        self.question = (PaperReviewQuestion.query.with_parent(self.event)
+                         .filter_by(id=request.view_args['question_id'])
+                         .one())
+
+
+class RHEditReviewingQuestion(RHManageQuestionBase):
+    def _process(self):
+        defaults = FormDefaults(obj=self.question, **self.question.field_data)
+        form = self.question.field.create_config_form(obj=defaults)
+        if form.validate_on_submit():
+            edit_reviewing_question(self.question, form)
```

Comment:
`update_reviewing_question` would be a better name. editing is usually what the user does, not the process of actually updating something